.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Linked issues
+
+Resolves: # (issue number)
+
+## Details
+
+Provide a list of your changes here. If you are fixing a bug, please provide steps to reproduce the bug.
+
+#### Change details
+
+> Describe your changes, with screenshots and code snippets as appropriate
+
+**code snippets**:
+
+**screenshots**:
+
+## Attestation Checklist
+
+- [ ] My code follows the style guidelines of this project
```

Comment:
If someone opens a PR for `Javascript`, who is responsible for creating the subsequent `DOTNET` and `Python` PR's?

Reply:
Once the JavaScript implementation is in place, new work items will need to be filed, either manually or by creating another workflow